lib/index.html

```diff
@@ -29,8 +45,60 @@
           });
         }
       }
+
+      function drawResult(test) {
```

Comment:
I know this is a copy/paste but can you adapt it to follow our code conventions. I know, there is no linter for this project but I will leave you some clues. :smile: